Docs/source/running_python/how_to_run.rst

```diff
@@ -8,6 +8,8 @@ WarpX can be run in one of two modes. It can run as a preprocessor, using the
 Python input file to generate an input file to be used by the C++ version, or
 it can be run directly from Python.
 
+In either mode, if using a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`__, be sure to activate it.
```

Comment:
This is just a little detail, I think you explained it already very clearly, but I would probably add:
```suggestion
In either mode, if using a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`__, be sure to activate it before compiling and running WarpX.
```

Reply:
Sounds good. Thanks!